src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -1546,7 +1546,8 @@ private void updateNotifications() {
 			String dateFormatted = pendingNotifs[i].getDateTime().get(Calendar.DAY_OF_MONTH) + " "
 					+ pendingNotifs[i].getDateTime().getDisplayName(Calendar.MONTH, Calendar.LONG,
 							Locale.getDefault())
-					+ " at " + pendingNotifs[i].getDateTime().get(Calendar.HOUR) + " "
+					+ " at " + pendingNotifs[i].getDateTime().get(Calendar.HOUR) + ":" 
+					+ pendingNotifs[i].getDateTime().get(Calendar.MINUTE)+ " "
```

Comment:
Please make the spacing consistent.  You should have a space before the plus sign.  CheckStyle should have complained about that.